scripts/v1alphaX/provision_cluster.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash
 
-METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
 # shellcheck disable=SC1091
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/common.sh"
-
 # shellcheck disable=SC1091
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/network.sh"
-
 # shellcheck disable=SC1091
 # shellcheck disable=SC1090
 source "${METAL3_DIR}/lib/images.sh"
 
-make_v1alphaX_machine controlplane "${CAPI_VERSION}" | kubectl delete -n metal3 -f -
-
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+   -e "metal3_dir=$SCRIPTDIR" \
+   -e "IMAGE_OS=$IMAGE_OS" \
+   -e "DEFAULT_HOSTS_MEMORY=$DEFAULT_HOSTS_MEMORY" \
+   -e "CAPI_VERSION=$CAPI_VERSION" \
```

Comment:
Isn't the variables in the playbook defaulting to the environment variable content ?